pysat/instruments/pysat_testing2d_xarray.py

```diff
@@ -24,11 +24,25 @@ def init(self):
 def load(fnames, tag=None, sat_id=None):
     
     # create an artifical satellite data set
+    '''
     parts = os.path.split(fnames[0])[-1].split('-')
     yr = int(parts[0])
     month = int(parts[1])
     day = int(parts[2][0:2])
     date = pysat.datetime(yr,month,day)
+    '''
```

Comment:
@rstoneback , is it good to hardcode a year here?

Reply:
The testing file generates its own fake filenames in the list_files routine. In this case I think it is ok since we have control over changing the simulated filenames. Open to discussion.